setup.py

```diff
@@ -9,7 +9,7 @@
 
 dependencies = [
     'beautifulsoup4>=4.6.3',
-    'lxml>=3.2.3,<4.4.0',
+    'lxml>=4.5.0',
```

Comment:
Looking at the other forks pycaption forks, I this requirement is relaxed simply as `lxml>=3.2.3` --- no need to push for the latest. Let other packages figure which exact version needed.